src/sentry/tasks/post_process.py

```diff
@@ -216,10 +218,15 @@ def process_snoozes(group):
     """
     from sentry.models import GroupSnooze, GroupStatus
 
-    try:
-        snooze = GroupSnooze.objects.get_from_cache(group=group)
-    except GroupSnooze.DoesNotExist:
-        return False
+    key = "postprocess_gs:1:%s" % (group.id)
+    snooze = cache.get(key)
+    if not snooze:
+        try:
+            snooze = GroupSnooze.objects.get(group=group)
+            cache.set(key, snooze, 3600)
```

Comment:
Is this setting the cache the a longer time than it was before? Potentially means we wait up to an hour to unsnooze a Group